monochart/values.yaml

```diff
@@ -143,6 +143,7 @@ deployment:
     # command:
     args: []
     lifecycle: {}
+    #hostPID: true
```

Comment:
In values files, don't you usually show the default value? I thought that was kind of the community standard, but could be misremembering.

Reply:
> In values files, don't you usually show the default value? I thought that was kind of the community standard, but could be misremembering.

Good call, I changed it. Retested with and without successfully.